java/src/main/native/src/ColumnVectorJni.cpp

```diff
@@ -19,6 +19,7 @@
 #include <arrow/api.h>
 #include <cudf/column/column_factories.hpp>
 #include <cudf/concatenate.hpp>
+#include <cudf/copying.hpp>
```

Comment:
Why?

Reply:
Because ColumnViewJni.cpp is using methods from cudf/copying.hpp (e.g.: cudf::copy_if_else) and should have included it all along. It was implicitly relying on the inclusion of cudf/copying.hpp from cudf_jni_apis.hpp (and now that implicit include is gone).